src/predictability.md

```diff
@@ -141,7 +141,7 @@ specifically to accommodate smart pointers, and so the traits should be used
 with those rules in mind.
 
 As a general guideline, consider the "_as-a_" rule. If you implement `Deref`,
-wherever `&Target` is acceptable, `&Self` works just as well as a `&Target`.
+wherever `&Target` is acceptable, `&Self` works just as well as `&Target`.
```

Comment:
I originally wanted to have a literal "as a" here, but I'd rather completely avoid the a/an question for `&Type` since this is still grammatically correct without the article.